docs/reference/commandline/swarm_join_token.md

```diff
@@ -65,6 +66,12 @@ SWMTKN-1-3pu6hszjas19xyp7ghgosyx9k8atbfcr8p2is99znpy26u2lkl-b30ljddcqhef9b9v4rs7
 ### `--rotate`
 
 Update the join token for a specified role with a new token and print the token.
+Invalidate the old token. Rotating a token does not effect existing nodes in the
+swarm.
+
+Because the token is a secret used to access the swarm, you should rotate the
```

Comment:
Actually replace the whole thing by:

`Because the token is a secret used to join the swarm, you might want to periodically rotate it, ensuring that any unknown token leaks will not allow a rogue node from joining the cluster.`